jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -169,18 +170,29 @@ function set_timer_defaults {
 
     local url=$(find_env "${prefix}_URL")
     url=$(find_env "${prefix}_XA_CONNECTION_PROPERTY_URL" "${url}")
-    if [[ $EJB_TIMER_DRIVER = *"mysql"* ]]; then
+    # Default to the Mariadb property
+    enabledTLSParameterName="enabledSslProtocolSuites"
+    if [[ $EJB_TIMER_DRIVER =~ mysql|mariadb ]]; then
+        if [[ $EJB_TIMER_DRIVER = *"mysql"* ]]; then
+            enabledTLSParameterName="enabledTLSProtocols"
+        fi
+
         if [ "x${url}" != "x" ]; then
-            EJB_TIMER_XA_CONNECTION_PROPERTY_URL="${url}?pinGlobalTxToPhysicalConnection=true"
+            EJB_TIMER_XA_CONNECTION_PROPERTY_URL="${url}?pinGlobalTxToPhysicalConnection=true\&amp;${enabledTLSParameterName}=${MYSQL_ENABLED_TLS_PROTOCOLS:-TLSv1.2}"
+            eval ${prefix}_URL="${url}?${enabledTLSParameterName}=${MYSQL_ENABLED_TLS_PROTOCOLS:-TLSv1.2}"
```

Comment:
@spolti we don't need "pinGlobalTxToPhysicalConnection=true" here also?

Reply:
no, the Pin.... property is intended to be used only for XA, the prefix_URL is for nonxa datasoure.

Reply:
And we only use XA for the timer, okay.